Makefile

```
@@ -22,32 +26,32 @@ compile:
 	@echo "==> Building the project"
 	@env CGO_ENABLED=0 GOCACHE=/tmp/go-cache GOOS=linux GOARCH=amd64 go build -a -o cmd/oras-csi-plugin/${NAME} cmd/oras-csi-plugin/main.go
 
-push-dev: build-dev
+push-dev: build-dev ## Build and push images to devtag
```

To be consistent with the next one.
```suggestion
push-dev: build-dev ## Build and push images to DEVTAG
```